app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -44,36 +43,43 @@ import com.wire.android.ui.theme.wireDimensions
 @Composable
 fun OngoingCallScreen(
     lifecycleOwner: LifecycleOwner = LocalLifecycleOwner.current,
-    ongoingCallViewModel: OngoingCallViewModel = hiltViewModel()
+    ongoingCallViewModel: OngoingCallViewModel = hiltViewModel(),
+    sharedCallingViewModel: SharedCallingViewModel = hiltViewModel()
 ) {
-    OngoingCallContent(ongoingCallViewModel)
-    observeScreenLifecycleChanges(lifecycleOwner, ongoingCallViewModel)
+    OngoingCallContent(sharedCallingViewModel)
+    observeScreenLifecycleChanges(lifecycleOwner, sharedCallingViewModel)
```

Comment:
minor thing: but what do tyou tihnk about passing a lambda to observeScreenLifecycleChanges like onStop() instead of sharedCallingViewModel so that we do not have to dive into observeScreenLifecycleChanges to see what we do with the viewmodel ?